lang/nl.json

```diff
@@ -13,73 +13,73 @@
   "introduction.name": "Introductie",
   "introduction.tagline": "...en waarom het belangrijk is",
   "introduction.copy": [
-    "Bij het nadenken over de natuur en het leven van wilde dieren, stellen we ons vaak dolfijnen voor die in de oceaan zwemmen of olifanten en leeuwen die in de savanne rondzwerven. We beelden ons meestal grote volwassen dieren in die een leven van vrijheid leiden. Deze rooskleurige kijk op de natuur komt echter niet eens in de buurt van hoe het leven van de meeste wilde dieren eruitziet.\", 'In werkelijkheid wordt het leven van de meeste wilde dieren niet gekenmerkt door vrijheid, maar door strijd. Ze hebben constant te maken met zwaar weer, hongersnood, ziekte en verwondingen, en net als een hond of een mens zouden lijden in hun positie, lijden zij ook."
+    "Als we nadenken over de natuur en het leven van wilde dieren, denken we al snel aan dieren als dolfijnen die speels in de oceaan zwemmen of olifanten en leeuwen die in de savanne rondzwerven. We beelden ons meestal grote volwassen dieren in, die een leven van vrijheid leiden. Deze rooskleurige blik op de natuur komt echter niet echt in de buurt van hoe het leven van de meeste wilde dieren er in de realiteit uit ziet. In werkelijkheid wordt het leven van de meeste wilde dieren niet gekenmerkt door vrijheid, maar door strijd. Ze hebben constant te maken met zwaar weer, hongersnood, ziekte en verwondingen. Net zoals een hond of een mens zouden lijden in hun positie, lijden ook zij."
```

Comment:
Shouldn't this be "Net zoals een hond of een mens zou* lijden in zijn/haar* positie"?

Reply:
Sounds right yeah! :)